build_msvc/msvc-autogen.py

```diff
@@ -17,6 +17,7 @@
     'libbitcoin_wallet_tool',
     'libbitcoin_wallet',
     'libbitcoin_zmq',
+    'bench_bitcoin'
```

Comment:
Nit: End with `,` to avoid touching this line next time a new item is added to the end of the `libs` list?